includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+class DesignSystemGlobalNavigationModel extends WikiaModel {
+	const DEFAULT_LANG = 'en';
+
+	private $hrefs = [
+		'default' => [
+			'fandom-logo' => 'http://fandom.wikia.com',
+			'games' => 'http://fandom.wikia.com/games',
+			'movies' => 'http://fandom.wikia.com/movies',
+			'tv' => 'http://fandom.wikia.com/tv',
+			'explore-wikis' => 'http://fandom.wikia.com/explore',
+			'community-central' => 'http://community.wikia.com/wiki/Community_Central',
+			'fandom-university' => 'http://community.wikia.com/wiki/Wikia_University',
+			'user-signin' => 'https://www.wikia.com/signin',
+			'user-register' => 'https://www.wikia.com/register',
+		],
+		'en' => [ ]
+	];
+
+	private $wikiId;
+	private $lang;
+
+	public function __construct( $wikiId, $lang = self::DEFAULT_LANG ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)